bundle/Controller/SearchController.php

```diff
@@ -4,14 +4,25 @@
 
 namespace Netgen\Bundle\SiteBundle\Controller;
 
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use Netgen\Bundle\SiteBundle\Core\Search\SuggestionResolver;
 use Netgen\EzPlatformSiteApi\Core\Site\Pagination\Pagerfanta\FindAdapter;
 use Pagerfanta\Pagerfanta;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
-use function trim;
```

Comment:
Why this change? You probably have an older version of PHP CS Fixer that does not understand the rule for this.

Reply:
@emodric I haven't used the CS fixer,  I've commited that accidentally :) Fixed in https://github.com/netgen/site-bundle/pull/14/commits/e903d27b8ba5c72ca09b3e350b37157ed34c5070.